lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -507,13 +507,12 @@ defmodule Kernel.SpecialForms do
   defmacro alias(module, opts), do: error!([module, opts])
 
   @doc """
-  Requires a given module to be compiled and loaded.
+  Tells Elixir you want to use the macros defined in a module.
```

Comment:
The structure I have in mind is:

1. Opt-in to use macros
2. Expand with example etc.
3. (Maybe) BTW, Elixir technically needs to compile and load the module (just so you know it in case you are suprised seen the code being run).